projects/angular/src/data/datagrid/built-in/filters/datagrid-string-filter.ts

```diff
@@ -72,6 +78,7 @@ export class DatagridStringFilter<T = any>
    */
   @ViewChild(ClrDatagridFilter) filterContainer: ClrDatagridFilter<T>;
 
+  filterLabel = '';
```

Comment:
```suggestion
  labelValue = '';
```

There is a `label` property and a `filterLabel` property, and the naming is a bit ambiguous. I recommend renaming this property to match `placeholderValue` so the the label and placeholder property naming is consistent.

Reply:
corrected in commit 42d1bf1